src/lib/layers/layers.component.ts

```diff
@@ -1,15 +1,38 @@
-import { Component } from '@angular/core';
+import { Component, Input, SimpleChanges, OnChanges, OnInit, HostBinding } from '@angular/core';
+import { SizeProp } from '@fortawesome/fontawesome-svg-core';
+import { faLayerClassList } from '../shared/utils/classlist.util';
+import { FaProps } from '../public_api';
```

Comment:
Please change import to use `../shared/models/props.model`. Public API guard is not intended to be imported from.